pronotepy/dataClasses.py

```diff
@@ -634,6 +695,38 @@ def to_dict(
         )
 
 
+class Comment(Object):
```

Comment:
I'm not sure if this class is completely necessary. I know the comment has an ID but we don't currently use it, so I think its a little unnecessary. I'd say just extract the text directly as a `str`.